<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>690</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="ui_central_wid">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="ui_main_frm">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="ui_btn_frm">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="ui_refresh_btn">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Refresh&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_lookup_btn">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Download&amp;nbsp;Metadata&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_rename_btn">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Rename Files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_clear_btn">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Clear&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Movies</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="uiMoviesTXT">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="uiMoviesBrowseBTN">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TV Shows</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="uiTVTXT">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="uiTVBrowseBTN">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ui_settings_btn">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ui_tree_frm">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="ui_main_split">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="handleWidth">
             <number>10</number>
            </property>
            <widget class="QFrame" name="ui_file_frm">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QTreeWidget" name="ui_file_tree">
                <property name="dragDropMode">
                 <enum>QAbstractItemView::DropOnly</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
                <property name="rootIsDecorated">
                 <bool>false</bool>
                </property>
                <column>
                 <property name="text">
                  <string>Filepath</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Renamed Filepath</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QFrame" name="ui_searchinfo_frm">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QSplitter" name="ui_searchinfo_split">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="handleWidth">
                 <number>10</number>
                </property>
                <widget class="QFrame" name="ui_search_frm">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="ui_manual_frm">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="ui_manual_title_txt">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="ui_manual_year_spin">
                       <property name="minimumSize">
                        <size>
                         <width>60</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="minimum">
                        <number>1800</number>
                       </property>
                       <property name="maximum">
                        <number>2100</number>
                       </property>
                       <property name="value">
                        <number>2000</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ui_manual_btn">
                       <property name="minimumSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>...</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="ui_searchtxt_frm">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="ui_search_txt">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="placeholderText">
                        <string>Search Title</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ui_set_movie_btn">
                       <property name="minimumSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>...</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTreeWidget" name="ui_search_tree">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                    <property name="rootIsDecorated">
                     <bool>false</bool>
                    </property>
                    <column>
                     <property name="text">
                      <string>Title</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Year</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Type</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QFrame" name="ui_episode_frm">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="ui_episode_btn_frm">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Season</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="ui_season_ddl">
                       <property name="minimumSize">
                        <size>
                         <width>80</width>
                         <height>30</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QToolButton" name="ui_set_episode_btn">
                       <property name="minimumSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>...</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTreeWidget" name="ui_episode_tree">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                    <property name="rootIsDecorated">
                     <bool>false</bool>
                    </property>
                    <property name="sortingEnabled">
                     <bool>true</bool>
                    </property>
                    <column>
                     <property name="text">
                      <string>Season</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Episode</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Title</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QFrame" name="ui_poster_frm">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="ui_poster_lyt">
                  <property name="margin">
                   <number>0</number>
                  </property>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="ui_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="ui_help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ui_about_act"/>
    <addaction name="ui_license_act"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="ui_help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ui_about_act">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="ui_license_act">
   <property name="text">
    <string>License</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
